File: app/src/main/java/com/afollestad/photoaffix/data/PhotoHolder.java
package com.afollestad.photoaffix.data;

import java.io.Serializable;

/** @author Aidan Follestad (afollestad) */
public class PhotoHolder implements Serializable {

  public Photo[] photos;

  public PhotoHolder() {}

  public PhotoHolder(Photo[] photos) {
    this.photos = photos;
  }
}

File: engine/src/main/java/com/afollestad/photoaffix/engine/photos/PhotoHolder.kt
/*
 * Licensed under Apache-2.0
 *
 * Designed and developed by Aidan Follestad (@afollestad)
 */
package com.afollestad.photoaffix.engine.photos

import java.io.Serializable

/** @author Aidan Follestad (afollestad) */
data class PhotoHolder(
  val photos: List<Photo>
) : Serializable
